lib/index.js

```diff
@@ -352,10 +366,10 @@ internals.getExistsValues = function (type, exists) {
             return (value.isContext ? '$' : '') + value.key;
         }
 
-        return value;
+        return JSON.stringify(value);
```

Comment:
Not if the method above returns a string for key references and strings for string literals. The whole map function could be moved into a helper, but it doesn't seem like a huge win one way or the other and requires that people duplicate the helpers for their own templates (if my understanding of how the `helpersPath` feature works is accurate).